### ISSUE TITLE
Add code coverage to test results

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/BuildTools#671]
Depends on https://github.com/Microsoft/vstest/issues/800, which should be resolved in VS 15.8

**Comment by ryanbrandenburg** [comment]
Can we see about triaging this next week? It would be really useful to know where things are at.

**Comment by natemcmaster** [comment]
Depends on #710

**Comment by natemcmaster** [comment]
This appears to be low priority and we don’t even have good tooling for this yet, especially if we are going to move to a distributed test system. Closing.
